store/gcworker/gc_worker.go

```diff
@@ -317,6 +371,37 @@ func (w *GCWorker) leaderTick(ctx context.Context) error {
 		return errors.Trace(err)
 	}
 
+	// Do keyspace delete range
+	if w.store.GetCodec().GetKeyspace() != nil {
```

Comment:
How about separating this keyspace logic into another function as it's pretty different from the original leader worker logic? Or we use different workers to distinguish the original worker type and the keyspace worker type, that may need more change or refactor.